### ISSUE TITLE
it seems that check_wandb_resume don't support multiple input files of images.

### ISSUE THREAD

**Issue by murphypei** [ultralytics/yolov5#2716]
it's just a minor bug about compatibility

## 🐛 Bug
for form2) or 3) in dataset yaml
```
# train and val data as 1) directory: path/images/, 2) file: path/images.txt, or 3) list: [path1/images/, path2/images/]
```

The `data_dict['train']`  and `data_dict['val']` are list in here.
```
if data_dict['train'].startswith(WANDB_ARTIFACT_PREFIX):
    ...

if data_dict['val'].startswith(WANDB_ARTIFACT_PREFIX):
   ...

```

## To Reproduce (REQUIRED)

Input:
```
train:
  [
    /path/image_list1.txt,
    /path/image_list2.txt,
    /path/image_list3.txt
  ]
```

Output:
```
YOLOv5 🚀 v4.0-185-g74276d5 torch 1.7.1+cu101 CUDA:0 (GeForce RTX 2080 Ti, 10989.4375MB)
                                             CUDA:1 (GeForce RTX 2080 Ti, 10989.4375MB)
                                             CUDA:2 (GeForce RTX 2080 Ti, 10989.4375MB)
                                             CUDA:3 (GeForce RTX 2080 Ti, 10989.4375MB)

    if data_dict['train'].startswith(WANDB_ARTIFACT_PREFIX):
AttributeError: 'list' object has no attribute 'startswith'
```

## Environment

 - OS: [Ubuntu]
 - GPU [2080 Ti]

**Comment by glenn-jocher** [comment]
@murphypei thanks for the bug report! 

@AyushExel this is true, the dataset yaml has 3 options for specifying train and val sets. The base values can be either 1) a *.txt file of image paths, or 2) a directory of images. Then 1 or 2 can be used by themselves (just one path, just one directory), or in a list (multiple *.txt files, multiple directories, or both).

This makes it easy for organizations to train on multiple datasets or multiple collects without having to merge the underlying data, i.e.:

```yaml
train: [path/to/dir1, path/to/dir2, path/to/dir3, ...]
```

**Comment by AyushExel** [comment]
@glenn-jocher @murphypei thanks for reporting this. I didn't know about the 3rd format of the dataset, might've missed it. I'll fix this right now.

**Comment by AyushExel** [comment]
@glenn-jocher  #2719 Should fix this issue. I tested it on the following dataset format and all operations including resume worked fine.
```
train: [../coco128/images/train2017/ , ../coco128/images/train20172/] # 128 images
val: [../coco128/images/train2017/ , ../coco128/images/train20172/] # 128 images
```
@murphypei let me know if the problem still exists after this is merged

**Comment by glenn-jocher** [comment]
@AyushExel thanks for the quick fix! I've merged PR #2719 now.

@murphypei please git pull to receive this update and let us know if you have any other issues.

**Comment by murphypei** [comment]
@AyushExel @glenn-jocher thanks, I have tested it without any issues.
